docs/languages/csharp/delegates-strongly-typed.md

```diff
@@ -1,11 +1,96 @@
 # Strongly Typed Delegates
 
+By [Bill Wagner](https://github.com/BillWagner)
+
 [Previous](delegate-class.md)
 
-This article will cover the classes in the .NET framework
-that support delegates, and how those map to the `delegate`
-keyword.
+In the previous article, you saw that you create specific delegate
+types using the `delegate` keyword. 
+
+Delegates are much more useful by embracing and enforcing type
```

Comment:
Delegates are much more useful than what, function pointers?  Or do you mean that they are more useful than they would be without type safety?  I'd rephrase.  Maybe "Delegates are useful also because they enforce type safety..."